components/SocialMedia/SocialMediaIconsList.tsx

```diff
@@ -1,16 +1,19 @@
-import { FC } from "react";
-import { IconContext } from "react-icons";
-import { FaDiscord, FaGithub, FaTwitter } from "react-icons/fa";
-import { GitHubForkButton } from "components/ForkButton/GitHubForkButton";
-
+import { FC } from 'react'
+import { IconContext } from 'react-icons'
+import { FaDiscord, FaGithub, FaTwitter } from 'react-icons/fa'
+import { GitHubForkButton } from 'components/ForkButton/GitHubForkButton'
```

Comment:
Everything looks great, but why was the code style of the document changed? Shouldn't that be a separate change/pr?